uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -1323,4 +1331,28 @@ private static String getSrc(String pipeline) {
     return srcDst[1];
   }
 
+  public void disableAutoScaling() {
```

Comment:
It's unnecessary to add enable/disable function, use setxxx shoud be enough.
ie: 
setAutoScaling(Boolean autoScaling)
setAutoBalancing(Boolean autoBalancing)